scripts.py

```diff
@@ -654,27 +648,6 @@ def process_patron(self, patron):
             if self.delete:
                 self._db.delete(credential)
 
-
-class BibliographicCoverageProvidersScript(RunCoverageProvidersScript):
```

Comment:
This script wasn't used or tested, and it was importing classes that were moved, so now is a good time to get rid of the code rather than try and get it working again.